.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -23,3 +23,10 @@ labels: feature
 
 ### Design Requirements
 <!-- If the customization includes input from our design team, the detailed requirements will be collected here. Note: These will exist mainly in German to simplify internal communication. -->
+
+#### Link to Figma
+<!-- The UI/UX team will post the link to figma here -->
```

Comment:
```suggestion
<!-- Link to the proposed design for this issue. Provided by the UI/UX team of the organization. -->
```
I would prefer a clear description of this issue section and not of our internal design process. This might be confusing of external contributors  as they don't know our design team...